tests/utils.py

```diff
@@ -167,15 +167,50 @@ def test_storage() -> FileStorage:
 
 
 @pytest.fixture(autouse=True)
-def autouse_test_storage() -> FileStorage:
-    return clean_test_storage()
+def autouse_test_storage(request) -> Optional[FileStorage]:
```

Comment:
this is maybe out of scope, but all of these fixtures need good names and docstrings and should be added to the docstring linter, I often look up what they do exactly in code if I need to disable something for testing. This one is a good example, if you never looked it up the name `autouse_test_storage` gives no real hint about what it does.

Reply:
right! we'll dedicate a week to refactor our utils and release them as OSS lib. version in dlt+ has nice docstrings btw.